data/common.yaml

```diff
@@ -75,6 +75,8 @@ unbound::log_local_actions: false
 unbound::log_servfail: false
 unbound::pidfile: '/var/run/unbound/unbound.pid'
 unbound::hints_file: "%{hiera('unbound::confdir')}/root.hints"
+unbound::manage_roothints_file: true
```

Comment:
instead of setting a default value here, can you please move that to init.pp? that allows puppet-strings to pick this up.